kuma/landing/urls.py

```diff
@@ -20,6 +21,19 @@
         name='promote_buttons'),
 ]
 
+if settings.MDN_CONTRIBUTION:
```

Comment:
The test for settings should move into the view (and return a 404), otherwise tests that use ``reverse('contribute')`` will fail when ``MDN_CONTRIBUTION`` is False.

Reply:
@michal-macioszczyk maybe you can look into this?